api/src/auth/drivers/ldap.ts

```diff
@@ -200,7 +199,10 @@ export class LDAPAuthDriver extends AuthDriver {
 				groupDn,
 				{
 					attributes: ['cn'],
-					filter: new EqualityFilter({ attribute: groupAttribute ?? 'member', value: userDn }),
+					filter: new EqualityFilter({
+						attribute: groupAttribute ?? 'member',
+						value: groupAttribute !== 'memberUid' ? userDn : identifier,
```

Comment:
So this won't work unless your "*_USER_IDENTIFIER" is also "uid" which for most people it won't be. To solve for this we'll need to add some code to the "fetchUserDn" function to also return the users "uid".

I want to make some bigger changes to how we get user info in this file. I can add the fix on to those if you're happy for me to do so?

Reply:
Sure, go ahead! Would love this to be working 👍